src/components/input/index.tsx

```diff
@@ -141,18 +141,22 @@ const Input = forwardRef<HTMLInputElement, Props>(
         name={name}
         value={value || ""}
         placeholder={placeholder || " "} // safari needs a non empty placeholder to make the :placeholder-shown pseudo selector work
-        className={classNames("w-12/12", {
-          // eslint-disable-next-line @typescript-eslint/naming-convention
-          input_withClearButton: hasClearButton,
-          // eslint-disable-next-line @typescript-eslint/naming-convention
-          "select_withSearchIcon bg-right bg-transparent bg-no-repeat": hasSearchIcon,
-          "floatingLabel-input": labelProps.floating,
-          "input_left hover:z-1 hover:transition hover:duration-200 focus:z-1":
-            position === "left",
-          "input_right -ml-1 hover:z-1 hover:transition hover:duration-200 focus:z-1":
-            position === "right",
-          "text-right": textAlignment === "right",
-        })}
+        className={classNames(
+          "w-12/12",
+          textAlignment === "center-right" ? "pl-45 pr-10 pb-3" : "px-10 py-13",
```

Comment:
This `px-10 py-13` is deleted from the base input above, correct? Is there no way to overwrite it here instead of deleting it from the base input? 

also it seems a bit weird, it seems as if the condition can be moved inside the brackets `{}`. We have this condition above the brackets, while `"text-right": textAlignment === "right",` is inside

Reply:
also `center-right` sounds a bit weird as well, from looking at it I'm not sure what it does

Reply:
I wanted to stick to the design but overcomplicated the task 👎 
Checked with Enrique who is fine having the unit as part of the label, so I removed this part 👍